seed_database/seedPostgreSQL.js

```diff
@@ -0,0 +1,61 @@
+const { Pool } = require('pg');
+const path = require('path');
+const pool = new Pool({
+  user: 'postgres',
+  host: '127.0.0.1',
+  database: 'sdcandrew',
+  password: '',
```

Comment:
would be nice to extract these config variables out

Reply:
I will look into using the dotenv package and a .env file to factor out these configuration variables!